code/components/extra-natives-five/src/VehicleExtraNatives.cpp

```diff
@@ -6,15 +6,19 @@
 */
 
 #include "StdInc.h"
+#include <unordered_set>
 #include <ScriptEngine.h>
 #include <atArray.h>
 
 #include <Local.h>
 #include <Hooking.h>
+#include <GameInit.h>
 
 #include <limits>
 #include <bitset>
 
+static std::unordered_set<fwEntity*> vehicles{};
```

Comment:
why the `{}`?

Reply:
it is list initialization

Reply:
not needed, default ctor will be equivalent